src/odemis/driver/orsay.py

```diff
@@ -2250,11 +2310,19 @@ def __init__(self, name, role, parent, **kwargs):
         # is imaged (i.e. doubling the imageFormat will double the resolution).
         # When the resolution is changed, the value of translation is adapted such that it is as close to its current
         # value as possible, whilst making sure the area defined by resolution completely fits the imageFormat.
-        self.imageFormat = model.VAEnumerated((1024, 1024), unit="px", choices={(512, 512), (1024, 1024)},
+        # The server accepts non-square image size, but we force it to be square,
+        # because it's mapped to scale, and the server always keeps the scale square.
+        self.imageFormat = model.ResolutionVA(SCANNER_MAX_RES, rng=(SCANNER_MIN_RES, SCANNER_MAX_RES),
                                               setter=self._imageFormat_setter)
-        self.resolution = model.TupleContinuous((1024, 1024), unit="px", range=[(1, 1), (1024, 1024)],
+        self.resolution = model.TupleContinuous(SCANNER_MAX_RES, unit="px", range=[(1, 1), SCANNER_MAX_RES],
                                                 setter=self._resolution_setter)
-        self.translation = model.TupleContinuous((0.0, 0.0), unit="px", range=[(-512.0, -512.0), (512.0, 512.0)],
+
+        # It uses the pixel convention: smallest value corresponds to top left of the image.
+        # The unit is the pixel at scale == 1.
+        # We accept floats, because it's Odemis convention, however, it will always be rounded
```

Comment:
I see you use "round" to round the float. I think it is nice to add that info also here so that a reader knows using which function the float is rounded to an int.